verizon/classes.py

```diff
@@ -27,105 +31,124 @@ def __init__(self, path, force = False):
             if vers != 0:
                 raise Exception(f"Unsupported repositoryformatversion : {vers}")
 
+
 class VerizonObject:
     def __init__(self, data=None) -> None:
-        if data != None:
+        if data is not None:
             self.deserialize(data)
         else:
             self.init()
-    
+
     def serialize(self, repo):
-        """ Read the objects contents, and do whatever it takes to convert it into a meaningful representation. """
+        """Read the objects contents, and do whatever it takes to convert it into a meaningful representation."""
         raise NotImplementedError
-    
+
     def deserialize(self, data):
         raise NotImplementedError
 
     def init(self):
         pass
 
+
 # Tree wrapper for a single record(a single path).
 class VerizonTreeLeaf:
     def __init__(self, mode, path, sha) -> None:
-        self.mode = mode 
+        self.mode = mode
         self.path = path
-        self.sha = sha 
+        self.sha = sha
 
-    
-## Type Header could be one of `blob`, `commit`, `tag`, `tree`.    
+
+## Type Header could be one of `blob`, `commit`, `tag`, `tree`.
 # Blobs are user data. The content of every file we put in git is stored as a blob.
 
+
 class VerizonBlob(VerizonObject):
-    fmt = b'blob'
+    fmt = b"blob"
 
     def serialize(self):
         return self.blobdata
-    
+
     def deserialize(self, data):
         self.blobdata = data
 
+
 class VerizonCommit(VerizonObject):
-    fmt = b'commit'
+    fmt = b"commit"
 
     def deserialize(self, data):
         self.kvlm = kvlm_parse(data)
-    
+
     def serialize(self, repo):
         return kvlm_serialize(self.kvlm)
-    
+
     def init(self):
         self.kvlm = dict()
 
+
 class VerizonTag(VerizonCommit):
-    fmt = b'tag'
+    fmt = b"tag"
+
 
 class VerizonTree(VerizonObject):
-    fmt = b'tree'
+    fmt = b"tree"
```

Comment:
I guess we can interchangeably switch between single and double quotes, didn't see the necessity to change all of single quotes to double - still, it works.